mslib/msui/mpl_qtwidget.py

```diff
@@ -94,6 +94,11 @@
     "plot_title_size": "default",
     "axes_label_size": "default"}
 
+mpl_logger = logging.getLogger('matplotlib')
```

Comment:
i would expect such blocks always after the import definition

and if it is always the same lines, can that become imported from an mslib.utils.loggerdef or other place?